recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
   sha256: 3f460b79d22a7fb1c9098ccacfe5da9da1558db4e96346bc847fde73c6cad6c0
 
 build:
-  number: 0
+  number: 1
```

Don't increment the build number for this
```suggestion
  number: 0
```